Netskope/netskope-transaction/ingest/parser.yml

```
@@ -0,0 +1,243 @@
+          - x-sr-ssl-handshake-error
+          - x-sr-ssl-client-certificate-error
+          - x-sr-ssl-malformed-ssl
+          - x-s-custom-signing-ca-error
+          - x-cs-ssl-engine-action
+          - x-cs-ssl-engine-action-reason
+          - x-sr-ssl-engine-action
+          - x-sr-ssl-engine-action-reason
+          - x-ssl-policy-src-ip
+          - x-ssl-policy-dst-ip
+          - x-ssl-policy-dst-host
+          - x-ssl-policy-dst-host-source
+          - x-ssl-policy-categories
+          - x-ssl-policy-action
+          - x-ssl-policy-name
+          - x-cs-ssl-version
+          - x-cs-ssl-cipher
+          - x-sr-ssl-version
+          - x-sr-ssl-cipher
+          - x-cs-src-ip-egress
+          - x-s-dp-name
+          - x-cs-src-ip
+          - x-cs-src-port
+          - x-cs-dst-ip
+          - x-cs-dst-port
+          - x-sr-src-ip
+          - x-sr-src-port
+          - x-sr-dst-ip
+          - x-sr-dst-port
+          - x-cs-ip-connect-xff
+          - x-cs-ip-xff
+          - x-cs-connect-host
+          - x-cs-connect-port
+          - x-cs-connect-user-agent
+          - x-cs-url
+          - x-cs-uri-path
+          - x-cs-http-version
+          - rs-status
+          - x-cs-app-category
+          - x-cs-app-cci
+          - x-cs-app-ccl
+          - x-cs-app-tags
+          - x-cs-app-suite
+          - x-cs-app-instance-id
+          - x-cs-app-instance-name
+          - x-cs-app-instance-tag
+          - x-cs-app-activity
+          - x-cs-app-from-user
+          - x-cs-app-to-user
+          - x-cs-app-object-type
+          - x-cs-app-object-name
+          - x-cs-app-object-id
+          - x-rs-file-type
+          - x-rs-file-category
+          - x-rs-file-language
+          - x-rs-file-size
+          - x-rs-file-md5
+          - x-rs-file-sha256
+          - x-error
+          - x-c-local-time
+          - x-policy-action
+          - x-policy-name
+          - x-policy-src-ip
+          - x-policy-dst-ip
+          - x-policy-dst-host
+          - x-policy-dst-host-source
+          - x-policy-justification-type
+          - x-policy-justification-reason
+          - x-sc-notification-name
+        delimiter: " "
+
+  - name: set_fields
+
+stages:
+  set_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: ["network"]
+          event.type: ["info"]
+
+          "@timestamp": "{{ parse_event.message.get('x-cs-timestamp') | to_iso8601 }}"
+
+      - set:
+          user.email: "{{ parse_event.message.get('cs-username')}}"
+        filter: '{{ parse_event.message.get("cs-username") | re_match("^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$") }}'
+
+      - set:
+          user.email: "{{ parse_event.message.get('x-cs-app-from-user')}}"
+        filter: '{{ parse_event.message.get("x-cs-app-from-user") | re_match("^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$") }}'
+
+      - set:
+          user.application: "{{ parse_event.message.get('cs-username') }}"
```

My bad. I think that was `user.name` expected here
```suggestion
          user.name: "{{ parse_event.message.get('cs-username') }}"
```